app/components/trainees/timeline/view.rb

```diff
@@ -51,8 +51,14 @@ def state_change_audits
         end
       end
 
-      def state_change_title(value)
-        I18n.t("components.timeline.titles.#{Trainee.states.key(value)}")
+      def state_change_title(changes)
```

Comment:
looks really nice...

![image](https://user-images.githubusercontent.com/470137/110094154-6c906900-7d93-11eb-883e-6197287aa525.png)


I wonder should we use the same language to describe the state throughout

ie `Trainee submitted for TRN` vs `Trainee Pending TRN`

Reply:
Defer to @edwardhorsford on this one!

Reply:
These should describe the action taken rather than the result - so I think `submitted for TRN` is correct here.